core/data/src/main/java/uk/gov/gchq/gaffer/data/graph/function/IterableFunction.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.data.graph.function;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+// TODO To be removed after Koryphe 1.1.0
```

Comment:
Let's just get Koryphe 1.1.0 released before we merge this in. In the koryphe version, this should be have some javadoc.